### ISSUE TITLE
Configure markdown render size.

### ISSUE THREAD

**Issue by williamspatrick** [ggozad/oterm#58]
When the query returns a markdown section, that gets rendered as a sort of a sub-window.  Is there a way to configure how large this section is maximum?  Typically I'd like it to be larger than what the default seems to be.

**Comment by ggozad** [comment]
Sure, I also think it's kind of small as is.
What would be a sensible default for you? I don't really want to make it a "setting", would rather we find something that would mostly work.
At the moment it's set to 20 lines. I was thinking of setting it to 50. Would that do it for you?

**Comment by williamspatrick** [comment]
Yeah, that's fine. I think I'm having trouble with the scrolling also (on Linux with kitty terminal).

**Comment by ggozad** [comment]
Awesome! I have a branch that is not done yet where this is implemented.
With regards to the scrolling. Iirc somebody else was having trouble with kitty. I do not know how to go about fixing it (tried testing kitty on my Mac but did not have any issues. Please feel free to open a new issue on this. If you have information on how to reproduce or can somehow debug the issue it would definitely help :)
